client/src/views/files/handler/FileHandler.vue

```diff
@@ -248,6 +252,7 @@ async function _getFileInfo(
       }
       offset += readResult.value.byteLength;
     }
+    fileInfoId.value = statsResult.value.id;
```

Comment:
What's the point of this variable?

Reply:
Can't this be put inside the content info with the other data instead of using a separate variable?